lib/internal/validators.js

```diff
@@ -61,7 +61,7 @@ function parseFileMode(value, name, def) {
   }
 
   if (isUint32(value)) {
-    return value;
+    return 0xFFFFFFFF & value;
   }
```

Comment:
Did this to convert it into the corresponding negative number. Not very sure yet why this doesn't run for `2176057344`.

Reply:
I guess it failed because the value got modified when it passed the validator once and since the validator currently doesn't accept negative values, it got rejected the second time.